docs/content/v0.20/packaging.md

```diff
@@ -12,11 +12,14 @@ There is also a container image involved.
 
 ### Local copy of KubeStellar git repo
 
-Because of [a restriction in one of the code generators that we
+**NOTE**: Because of [a restriction in one of the code generators that
+we
 use](https://github.com/kubernetes/code-generator/blob/v0.28.2/kube_codegen.sh#L394-L395),
-a contributor who works on API changes needs to have their local copy
-of the git repo in a directory whose name ends with the Go package
-name --- that is, ends with `github.com/kubestellar/kubestellar/`.
+a contributor who works on API changes, or even just invokes
```

Comment:
Should we explain all of this or just recommend this path for all developers?  Especially we add "or even just invokes.."   without explaining why and when a developer may need to run this.

Reply:
Done.